tests/test_placeholder_validator.py

```diff
@@ -73,3 +81,43 @@ def test_placeholder_plurals():
     ]
 
     assert validator.errors == expected_errors
+
+
+def _open_and_load_schema_file(file):
+    with open(
+        os.path.join(os.path.dirname(os.path.realpath(__file__)), file), encoding="utf8"
+    ) as json_file:
+        return json.load(json_file)
```

Comment:
This function is defined in `test_questionnaire_validatory.py`, we should share between the tests.